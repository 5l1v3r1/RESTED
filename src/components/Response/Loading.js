import React from 'react';

import { LoadingSpinner } from './StyledComponents';

export default function Loading() {
  return (
    <LoadingSpinner
      icon="gear"
      className="fa-spin col-xs-12 text-center"
    />
  );
}

