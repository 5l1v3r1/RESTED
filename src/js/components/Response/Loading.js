import React from 'react';

import Fonticon from '../Fonticon';

export default function Loading() {
  return (
    <Fonticon
      icon="gear"
      className="fa-spin col-xs-12 text-center"
    />
  );
}

