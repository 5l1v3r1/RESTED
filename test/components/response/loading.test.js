import React from 'react';
import renderer from 'react-test-renderer';

/* eslint-disable import/no-unresolved */
import Loading from 'components/Response/Loading';

describe('Loading', () => {
  it('matches the previous snapshot', () => {
    const tree = renderer.create(
      <Loading icon="cog" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

