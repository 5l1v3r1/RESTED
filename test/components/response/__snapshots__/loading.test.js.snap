// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading matches the previous snapshot 1`] = `
<i
  className="fa fa-gear fa-spin col-xs-12 text-center"
  role="presentation"
/>
`;
